carbon-design-system/carbon-addons-iot-react
# feat(cardeditor): add maximumDataPoints to data series cards

Closes #

![image](https://user-images.githubusercontent.com/6663002/145124830-ccb6e344-663c-4b2e-87a4-f004d8bcd4f9.png)


**Summary**

- Support the ability to capture the maximumDataPoints a DataSeriesCard should show.  This is important because the TimeSeriesCards can reach scale issues once we get to a certain size

**Change List (commits, features, bugs, etc)**

- feat(DataSeriesFormSettings): add the property to capture maximumDataPoints for the card
- feat(DashboardEditor): support passing the 18n to upstream default cards 
- feat(index): export the getDefaultCard utility function so downstream consumers can use it in addition to their own getDefaultCard callbacks in the DashboardEditor

**Acceptance Test (how to verify the PR)**

- Verify that a TimeSeriesCard in the card editor has a new setting to capture the maximumDataPoints

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify the existing getDefaultCard callbacks still work



**Things to look for during review**
